pkg/kubectl/cmd/set/set_env.go

```diff
@@ -405,25 +410,33 @@ func (o *EnvOptions) RunEnv(f cmdutil.Factory) error {
 
 	allErrs := []error{}
 
+	p, matched, err := o.PrintFlags.ToPrinter("env updated", o.DryRun)
```

Comment:
@deads2k I'm setting the "operation" and "dryRun" values for the name printer here, but can also set the via a Complete() method for the name printer if that makes more sense:

```Go
// sets the values in o.NamePrinter.Operation and o.NamePrinter.DryRun
o.PrintFlags.Complete("env updated", o.DryRun)
```